signal-hook-registry/src/lib.rs

```diff
@@ -462,18 +462,31 @@ const FORBIDDEN_IMPL: &[c_int] = &[SIGKILL, SIGSTOP, SIGILL, SIGFPE, SIGSEGV];
 ///
 /// # Safety
 ///
-/// This function is unsafe, because the `action` is run inside a signal handler. The set of
-/// functions allowed to be called from within is very limited (they are called async-signal-safe
-/// functions by POSIX). These specifically do *not* contain mutexes and memory
-/// allocation/deallocation. They *do* contain routines to terminate the program, to further
-/// manipulate signals (by the low-level functions, not by this library) and to read and write file
-/// descriptors. Calling program's own functions consisting only of these is OK, as is manipulating
-/// program's variables ‒ however, as the action can be called on any thread that does not have the
-/// given signal masked (by default no signal is masked on any thread), and mutexes are a no-go,
-/// this is harder than it looks like at first.
-///
-/// As panicking from within a signal handler would be a panic across FFI boundary (which is
-/// undefined behavior), the passed handler must not panic.
```

Comment:
This info about panics still seem relevant, but isn't reflected in the new text. Should it still be there?